### PR TITLE
Add support for Django 3.2

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ jobs:
...
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9', 'pypy3']
-        django-version: ['2.2', '3.0', '3.1']
+        django-version: ['2.2', '3.0', '3.1', '3.2']
         # Tox configuration for QA environment
         include:
           - python-version: '3.8'
             django-version: 'qa'
-          # Django >= 3.2 only supports >= Python 3.8
+          # Django > 3.2 only supports >= Python 3.8
```
Comment (hramezani) [inline .github/workflows/test.yml:18]: Django3.2 supports Python >= 3.6 and Django4.0 will support Python >=3.8

File: axes/handlers/cache.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def reset_attempts(
             )
 
         cache_keys.extend(
-            get_client_cache_key(AccessBase(username=username, ip_address=ip_address))
+            get_client_cache_key(AccessAttempt(username=username, ip_address=ip_address))
```
Comment (hramezani) [inline axes/handlers/cache.py:47]: From Django3.2 release note:

> Instantiating an abstract model now raises TypeError.

File: axes/handlers/cache.py [hunk 3]
```diff
@@ -44,7 +44,9 @@ def reset_attempts(
             )
 
         cache_keys.extend(
-            get_client_cache_key(AccessBase(username=username, ip_address=ip_address))
+            get_client_cache_key(
+                AccessAttempt(username=username, ip_address=ip_address)
+            )
```
Comment (hramezani) [inline axes/handlers/cache.py:49]: From Django3.2 release note:

> Instantiating an abstract model now raises TypeError.

File: pyproject.toml [hunk 4]
```diff
@@ -37,6 +38,7 @@ deps =
     dj22: django>=2.2,<2.3
     dj30: django>=3.0,<3.1
     dj31: django>=3.1,<3.2
+    dj32: django>=3.2,<4.0
```
Comment (aleksihakli) [inline pyproject.toml:41]: Although the next release will be 4.0 this should probably be defined as:

    dj32: django>=3.2,<4.0

This is more consistent and avoids problems if plans change or tools get funny release versions. Has happened before, will happen again. Maybe not with Django, but a lot of places had lax version checks back when Python 3 became mainstream as well as when Django 3 was released and both produced a bunch of issues in libraries.

**Review by aleksihakli — APPROVED** [review]
LGTM! The only thing that caught my eye was the pyproject.toml tox definition.

**Pull request merged**